tests/Integration/PackagesListBuilderTest.php

```diff
@@ -24,131 +26,141 @@ public function testBase(): void
+                    ],
                 ],
             ],
         ]);
 
         $this->execComposer('require test/c');
         $this->assertMergePlan([
-            'params' => [
-                '/' => [
-                    'config/params.php',
-                    '?config/params-local.php',
-                ],
-                'test/c' => [
-                    'config/params.php',
-                ],
-                'test/a' => [
-                    'config/params.php',
-                ],
-            ],
-            'web' => [
-                '/' => [
-                    'config/web.php',
-                ],
-                'test/ba' => [
-                    'config/web.php',
-                ],
-                'test/c' => [
-                    'config/web.php',
+            Options::DEFAULT_ENVIRONMENT => [
+                'params' => [
+                    Options::DEFAULT_ENVIRONMENT => [
+                        'config/params.php',
+                        '?config/params-local.php',
+                    ],
+                    'test/c' => [
+                        'config/params.php',
+                    ],
+                    'test/a' => [
+                        'config/params.php',
+                    ],
                 ],
-                'test/a' => [
-                    'config/web.php',
+                'web' => [
+                    Options::DEFAULT_ENVIRONMENT => [
+                        'config/web.php',
+                    ],
+                    'test/ba' => [
+                        'config/web.php',
+                    ],
+                    'test/c' => [
+                        'config/web.php',
+                    ],
+                    'test/a' => [
+                        'config/web.php',
+                    ],
                 ],
             ],
         ]);
 
         $this->execComposer('require test/custom-source');
         $this->assertMergePlan([
-            'params' => [
-                '/' => [
-                    'config/params.php',
-                    '?config/params-local.php',
-                ],
-                'test/c' => [
-                    'config/params.php',
-                ],
-                'test/custom-source' => [
-                    'params.php',
-                ],
-                'test/a' => [
-                    'config/params.php',
-                ],
-            ],
-            'subdir' => [
-                'test/custom-source' => [
-                    'subdir/*.php'
-                ],
-            ],
-            'web' => [
-                '/' => [
-                    'config/web.php',
-                ],
-                'test/ba' => [
-                    'config/web.php',
-                ],
-                'test/c' => [
-                    'config/web.php',
+            Options::DEFAULT_ENVIRONMENT => [
+                'params' => [
+                    Options::DEFAULT_ENVIRONMENT => [
```

Comment:
Seems here `Options::DEFAULT_ENVIRONMENT` not environment. This is application files for config. May be create new constant for this?